docs/sections/writing_workflows/workfunction.md

```diff
@@ -30,8 +30,6 @@ $ verdi data core.structure import ase Si.cif
 Note that the output of `verdi data core.structure import` will probably show a different value for the PK of the structure node you just created: **make a note of this PK**, as you will need to replace it in code snippets later in this tutorial.
 You can also manually download the {{ download }} {download}`Si.cif <include/data/Si.cif>` structure file and copy it in your work environment instead of using `wget`.
 
-:::
-
```

Comment:
This was the stupid culprit of the CI errors? 🤣

Anyways, thanks for the fix @mbercx !

Reply:
Haha, _and_ the missing `Si.cif` file. 😜